h/accounts/__init__.py

```diff
@@ -16,9 +16,9 @@ class NoSuchUserError(Error):
     pass
 
 
-def make_admin(user_id):
+def make_admin(userid):
     """Make the given user an admin."""
-    user = models.User.get_by_username(user_id)
+    user = models.User.get_by_username(userid)
```

Comment:
"acct:foo@bar.com" is a `userid`, "foo" is a username. `get_by_username` accepts a username. This param name should probably change.